tools/build-cocoa.sh

```diff
@@ -79,6 +79,8 @@ done
 
 ln -s librealm-macosx.a core/librealm.a
 ln -s librealm-macosx-dbg.a core/librealm-dbg.a
+ln -s librealm-parser-macosx.a core/librealm-parser.a
+ln -s librealm-parser-macosx-dbg.a core/librealm-parser-dbg.a
```

Comment:
Creating this symlink isn't sufficient given that `librealm-parser-macosx{,-dbg}.a` aren't in the archive. I think the logic earlier in the script needs to be modified to extract and repackage the parser libraries from the archives like it does with the main core static library.